README.md

```diff
@@ -115,7 +115,7 @@ test, execute:
 This allows running integration tests that exercise the capabilities of AEM via
 HTTP calls to its API. To run the integration tests, run:
 
-    mvn clean verify -Plocal
+    mvn clean verify -Plocal,cloud
```

Comment:
Actually it's `cloud` or `classic` depending on what profile you build.

Reply:
Updated it, added two samples for cloud or classic